pywbem/cim_operations.py

```diff
@@ -9730,8 +9730,14 @@ def is_subclass(ch, ns, super_class, sub):
       sub:
         The subclass.  This can either be a string or a
         :class:`~pywbem.CIMClass` object.
-    """
 
+    Returns:
+        :class:`py:bool`: Boolean True if the assertion is True (sub is a
+        subclass of super_class) or False if it is not a subclass.
+
+    Raises:
+        CIMError if the the sub is not a valid class in the repo
+    """
```

Comment:
While it is a good idea to document the return value and exceptions, it is not in scope of what the PR says it does.

Reply:
Added to PR doc.

Reply:
I assume you did not want me to take the change out.  I tried to only add tests for a those parse_... methods that were reporting missing in coverage and to be honest, I am not sure if RETURNVALUE was in that catagory or not.